tools/buildifier/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@buildifier_prebuilt//:rules.bzl", "buildifier")
+
+buildifier(
```

Comment:
Does this enable all lints?

Reply:
I thought it did but it did not.  Setting lint_warnings=["all"] does it.